src/main/kotlin/ecommerce/service/PaymentService.kt

```diff
@@ -0,0 +1,57 @@
+package ecommerce.service
+
+import ecommerce.client.StripeClient
+import ecommerce.dto.PaymentIntentResponse
+import ecommerce.dto.PaymentRequest
+import ecommerce.exception.BadRequestException
+import ecommerce.exception.ExternalServiceException
+import ecommerce.model.Order
+import ecommerce.model.OrderStatus
+import ecommerce.repository.OrderRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class PaymentService(
+    private val stripeClient: StripeClient,
+    private val orderRepository: OrderRepository,
```

Comment:
Just like commented out code,
I think it would be nice to remove unused injection as well.

It would be great to check other classes too!